src/Parsers/ASTRenameQuery.h

```diff
@@ -94,7 +94,24 @@ class ASTRenameQuery : public ASTQueryWithOutput, public ASTQueryWithOnCluster
     ASTPtr clone() const override
     {
         auto res = std::make_shared<ASTRenameQuery>(*this);
-        res->cloneChildren();
```

Comment:
`cloneChildren` do not update elements, so we have old ASTs there, we should update them also in `clone`